version.py

```diff
@@ -196,6 +197,29 @@ async def update_npm_version(*, new_version, working_dir, readonly):
     return old_version
 
 
+async def update_version_file(*, new_version, working_dir, readonly):
+    """
+    Update a VERSION file.
+
+    Args:
+        new_version (str): The new version
+        working_dir (str): The directory of the library
+        readonly (bool): If true, return the old version if found but don't actually write changes to any files
+
+    Returns:
+        str:
+            The old version which has been successfully replaced with the new version.
+            On error, an exception will raise.
+    """
+    with open(Path(working_dir) / "VERSION", "r") as f:
+        old_version = f.readline()
+    if not readonly:
+        await check_output(
```

Comment:
Up to you but I don't think we need to use `sed` here since we can just write a new file with the version string